paasta_tools/setup_kubernetes_job.py

```diff
@@ -213,6 +214,12 @@ def setup_kube_deployments(
             for deployment in existing_kube_deployments
         }
 
+        existing_git_shas: Dict[Tuple[str, str, str], List[str]] = {}
+        for deployment in existing_kube_deployments:
+            existing_git_shas.setdefault(
+                (deployment.service, deployment.instance, deployment.namespace), []
+            ).append(deployment.git_sha)
```

Comment:
Thoughts on using DeploymentVersion (aka git sha+ image_version) instead of just git sha? We don't seem to have many using this right now but if we did, we'd probably want to prioritize those rebuilds over big bounces too?

Reply:
Probably easy enough.